action.py

```diff
@@ -113,7 +116,9 @@ def parse_video(self, video: Dict[str, Any]) -> str:
         if video_id in self._youtube_data:
             content_details = self._youtube_data[video_id]["contentDetails"]
             if self._show_duration:
-                params["duration"] = self.parse_iso8601_duration(content_details["duration"])
+                params["duration"] = self.parse_iso8601_duration(
+                    content_details["duration"]
+                )
```

Comment:
It seems some lines such as this one were auto-formatted. Preferably, it should be put back to reduce the number of unrelated changes in the PR.

The settings I used in VS Code are:

```json
"python.formatting.provider": "black",
"python.formatting.blackArgs": [
    "--line-length",
    "100"
],
```

You may be using it with a shorter line length (default 80) which could be causing the discrepancy.

At some point (in a later PR), it may be good to get some auto-formatter workflows running and possibly document the style settings used by the project.

Reply:
We could utilise the `poetry.tool.scripts` to have some shorthand commands to run black with the project's style settings.